entrypoint.sh

```diff
@@ -8,6 +8,11 @@ if [ -z "${JEKYLL_PAT}" ]; then
   exit 1
 fi 
 
+if [ -n "${INPUT_JEKYLL_ROOT}" ]; then
+  echo "::debug ::Switching to ${INPUT_JEKYLL_ROOT} as a root directory"
+  cd ${INPUT_JEKYLL_ROOT}
```

Comment:
Changing directory may not be good, since it changes the bundle install path of vendor/bundle. 
Rather, I think changing the `bundle install` command and adding a [`--gemfile` option](https://bundler.io/man/bundle-install.1.html#OPTIONS) would be better.

Reply:
I believe this would be intended behavior for the use case of e.g. Gradle-based Java project with Bundler-based Jekyll site isolated entirely in /docs subdirectory. But I see your point, this option should be useful for Ruby projects with Jekyll site in /docs. I believe these are a bit different workflows though, and I'm not sure one configuration option can cover both of these naturally.

If you'd like let's continue in #31 to not split efforts on the same feature to multiple places.